README.md

```diff
@@ -50,7 +50,7 @@ like this:
 
 ```groovy
 buildscript {
-    ext.jenkinsJobdslPluginVersion = '3.2.1'
+    ext.jenkinsJobdslPluginVersion = '3.2.3'
```

Comment:
Doesn't this then also need `compile 'org.codehaus.groovy:groovy-all:2.4.15'` in `dependencies` below?

Reply:
Edit: I just saw you did that later in https://github.com/heremaps/gradle-jenkins-jobdsl-plugin/pull/104/commits/1a5fc2de6ba82751ce6af18a6a9048cd9b041054. Shouldn't that commit be squashed to this one then?